### ISSUE TITLE
Unable to upload multiple large files (25 files) from the browser to S3 bucket

### ISSUE THREAD

**Issue by AlbionS** [aws/aws-sdk-js#3096]
HI,

I am trying to upload some files from a browser to S3.

![dropzone](https://user-images.githubusercontent.com/29225486/74419995-087aca00-4e4b-11ea-85b3-0f57520b22d6.PNG)

The problem that I have, is that some of the files pass correctly to S3 but some not. 

If I put 15 files at the same time the upload will stop and **I get completely disconnected form the Internet** !! **My wifi completely stops because of the upload and I dont understand why**. I even tried to upload files while I was connected via Ethernet and it was the same thing, during the upload my internet connection stops !! The upload is stopping my internet connection. Since I get disconnected form the internet and there is no internet, AWS SDK tries 3 more times and after it stops (normal cause I have no internet).

If I have small files or only one file at a time, it works fine most of the time. But sometimes as I said it stops my Internet connection and I have to wait 1,2 minutes until my connection is back, try it again (if I try again I will get the same problem) so I upload them 1 by 1 until I upload all of them.

**Is it because there are much files uploading at the same time and my internet stops ?
Or is there any other way to upload multiple files to S3 without having this problem ?**

My code to upload files dropped in the dropzone :
```
const s3 = new AWS.S3({
	region : "eu-west-3",
	accessKeyId : response.data.s3_access_key_id,
	secretAccessKey : response.data.s3_secret_access_key
});


Array.from(sourceDropzone.files).forEach(function(file){
	var basename = file.name;
        var key ="/Origin_files/"+basename;
        var params = {
            Bucket: response.data.bucketName, 
            Key: key,
            ACL: "private",
            Body: file
        };
        var options = {partSize: 10 * 1024 * 1024, queueSize: 1};
        s3.upload(params, options, function(err, data) {
            if(err){
                  // an error occurred 
                  }
                  else{
                   //SUCCESS
                  }
         }).on('httpUploadProgress', function (progress) {
                   console.log(progress);
          	});
});
```

**PS I have a fast internet connection**

**Comment by ajredniwja** [comment]
Hey @AlbionS, thank-you for reaching out to us with your issue.

I was not able to reproduce this, the SDK wont have any role in stopping the internet to work. Is there any more context you van provide or anything that you might be doing differently?

**Comment by AlbionS** [comment]
I found a solution. I think trying to upload all the files at the same time can stop the internet connection because I think S3 is waiting for the file but since there are lots of them than it  should expire and cause some internet problems (strange though).

So what I did is upload one file at a time and set AWS.config.httpOptions.timeout = 0 (even will low connection will work); and it is working perfectly fine. I am able to upload files correctly.
![image](https://user-images.githubusercontent.com/29225486/74822873-18405580-5306-11ea-89b4-4f000ba5432d.png)

Thanks  a lot for trying :)

Best regards,
Albion
